src/update_handler.c

```diff
@@ -1034,6 +1080,157 @@ static gboolean img_to_fs_handler(RaucImage *image, RaucSlot *dest_slot, const g
 	return res;
 }
 
+static gboolean img_to_boot_emmc_handler(RaucImage *image, RaucSlot *dest_slot, const gchar *hook_name, GError **error) {
+
+	gboolean res = FALSE;
+	int out_fd;
+	gint ret = 0;
+	gint part_active;
+	gint part_active_after;
+	GOutputStream *outstream = NULL;
+	GError *ierror = NULL;
+	RaucSlot *part_slot = NULL;
+
+	/* run slot pre install hook if enabled */
+	if (hook_name && image->hooks.pre_install) {
+		res = run_slot_hook(
+				hook_name,
+				R_SLOT_HOOK_PRE_INSTALL,
+				NULL,
+				dest_slot,
+				&ierror);
+		if (!res) {
+			g_propagate_error(error, ierror);
+			goto out;
+		}
+	}
+
+	/* read active boot partition from ext_csd */
+	ret = r_emmc_read_bootpart(dest_slot->device, &part_active, &ierror);
+	if (!ret) {
+		g_propagate_error(error, ierror);
+		res = FALSE;
+		goto out;
+	}
+	g_message("Found active eMMC boot partition %sboot%d",
```

Comment:
This printout will be wrong if `part_active` is 6.

Reply:
Yes, that's misleading. Will fix it.